inference-engine/thirdparty/clDNN/runtime/kernels_cache.cpp

```diff
@@ -411,23 +415,40 @@ void kernels_cache::build_all() {
         std::lock_guard<std::mutex> lock(_mutex);
         get_program_source(_kernels_code, &batches);
 #if (CLDNN_THREADING == CLDNN_THREADING_TBB)
-        int n_threads = _engine.configuration().n_threads;
-        arena = std::unique_ptr<tbb::task_arena>(new tbb::task_arena());
-        arena->initialize(n_threads);
+        const auto n_threads = _engine.configuration().n_threads;
+        const auto core_type = _engine.configuration().cpu_core_type;
+        _task_executor = std::unique_ptr<CPUStreamsExecutor>(new CPUStreamsExecutor(
+            IStreamsExecutor::Config{
+                "CLDNNPlugin executor for build kernels cache on load network", // name
+                1,                                                              // Number of streams to set number of stream, load_network has 1 stream
+                n_threads,                                                      // threadsPerStream used to set max_concurrency for task_arena
+                IStreamsExecutor::ThreadBindingType::HYBRID_AWARE,              // threadBindingType
+                1,                                                              // threadBindingStep used in ThreadBindingType::Cores
+                0,                                                              // threadBindingOffset used in ThreadBindingType::Cores
+                1,                                                              // Number of threads
```

Comment:
how is this different from the above n_threads field?

Reply:
- n_threads  is related threadsPerStreams which is to set max_concurrency for task_arena.
- _threads is Number of threads distributed between streams. but it is not used variable it is just reserved